addons/l10n_cl/data/template/account.tax-cl.csv

```diff
@@ -99,3 +99,11 @@
 "","","","","","","","","","","","tax","invoice","+IVA Pagado Compras Supermercado","account_110710","","",""
 "","","","","","","","","","","","base","refund","-Compras De Supermercado","","","",""
 "","","","","","","","","","","","tax","refund","-IVA Pagado Compras Supermercado","account_110710","","",""
+"iec_gasoline","IEC Gas","Specific Tax Gasoline","IEC Gasolina","1.0","percent","purchase","35","tax_group_impuestos_especificos","","5","base","invoice","+Compras Des Combustibles","","","Impuesto Específico Gasolina","IEC Gasolina"
+"","","","","","","","","","","","tax","invoice","+IEC Compras Des Combustibles","account_110760","","",""
+"","","","","","","","","","","","base","refund","-Compras Des Combustibles","","","",""
+"","","","","","","","","","","","tax","refund","-IEC Compras Des Combustibles","account_110760","","",""
+"iec_diesel","IEC Die","Specific Tax Diesel","IEC Diesel","1.0","percent","purchase","28","tax_group_impuestos_especificos","","5","base","invoice","+Compras Des Combustibles","","","Impuesto Específico Diesel","IEC Diesel"
+"","","","","","","","","","","","tax","invoice","+IEC Compras Des Combustibles","account_110760","","",""
+"","","","","","","","","","","","base","refund","-Compras Des Combustibles","","","",""
+"","","","","","","","","","","","tax","refund","-IEC Compras Des Combustibles","account_110760","","",""

```

Comment:
Also we would need a local upgrade script for that, check the same pr that i have sent you in the other comments. Also, if i'm not mistaken the new account won't be added for existing database, so we might need to add it with a post init or in migration :thinking: 
(@clbr-odoo right ?)